library/src/main/res/values/colors.xml

```diff
@@ -1,22 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <!--
-    Palette from
-    https://material.io/tools/color/#!/?view.left=0&view.right=0&primary.color=01579B&secondary.color=00E676
```

Comment:
Do you happen to have the palette from material.io?
If yes can I ask you to link it here as it's easy to link it afterwards :)

Reply:
Yes, I use the material one. Ok, will bring it back. As to night colors - I do according to guidelines and use the same colors as for day, but less contrast versions of them.

Reply:
Unfortunately, I can't link the palette, because material.io still has old style of colors naming (for example `primary_dark` instead or `primary_variant`), so it wouldn't be valid.
Here is an example of this palette: https://material.io/resources/color/#!/?view.left=0&view.right=0&primary.color=01579b&secondary.color=00e676&secondary.text.color=000000